paper-server/patches/sources/net/minecraft/world/level/EntityGetter.java.patch

```diff
@@ -1,5 +1,14 @@
 --- a/net/minecraft/world/level/EntityGetter.java
 +++ b/net/minecraft/world/level/EntityGetter.java
+@@ -56,7 +_,7 @@
+             return List.of();
+         } else {
+             Predicate<Entity> predicate = entity == null ? EntitySelector.CAN_BE_COLLIDED_WITH : EntitySelector.NO_SPECTATORS.and(entity::canCollideWith);
+-            List<Entity> entities = this.getEntities(entity, collisionBox.inflate(1.0E-7), predicate);
++            List<Entity> entities = this.getEntities(entity, collisionBox.inflate(1.0E-7), predicate); // Paper - Call HangingBreakByEntityEvent for collision with boats; diff on change
```

Comment:
You could use Util.COLLISION_EPSILON as a constant rather than adding a diff on change.